rl_baselines/train.py

```diff
@@ -158,13 +171,39 @@ def callback(_locals, _globals):
             printGreen("Saving new best model")
             ALGO.save(LOG_DIR + ALGO_NAME + "_model.pkl", _locals)
 
+        if n_episodes >= 0:
+
+            #For every checkpoint, we create one directory for saving logs file (policy and run mean std)
+            if n_episodes % EPISODE_WINDOW_DISTILLATION_WIN == 0:
+                ALGO.save(LOG_DIR + ALGO_NAME +'_' + str(n_episodes)+ "_model.pkl", _locals)
+                if(CROSS_EVAL):#If we want to do the cross evaluation after the training
+                    eps_path = LOG_DIR + "model_"+ str(n_episodes)
+                    try:
+                        os.mkdir(LOG_DIR + "model_"+ str(n_episodes))
+                    except OSError:
```

Comment:
Hi @sun-te, 
- It seems like the directory is already created using ALGO.save ?
 See the message I keep getting while learning RL with the script: 
`Creation of the directory logs/sc_test_gen/OmnirobotEnv-v0/supervised/ppo2/19-05-23_11h19_14/model_3800 failed` for several eps values
- Plus, why is there a folder model_N/ with models logs (*.pkl, *.rms) 
and additionally, an associated file ppo2_N_model.pkl ?

Reply:
- The reason why I finally choose to save the model into a folder is that we need to save the `ret_rms` and `obs_rms` as a normalization reference when evaluating the policy trained with `srl_combination`
- As for the failure during the creation of the folder, I think it is caused by multiprocessing, since multi-workers are trying to create a file with the same name. For example: 6 cpus are all training the agent at episode-60. When the training is done, only the cpu who firstly finished the tasks is able to create a log folder : model_3800

Reply:
If I don't save them in one folder: `ppo`(The model itself, weight matrix), 'ppo_model', 'ret_rms'(normalization file) 'obs_rms'(normalization file), there might be too many items in one folder